loadtest/control/actions.go

```diff
@@ -729,3 +729,33 @@ func Reload(u user.User) UserActionResponse {
 
 	return UserActionResponse{Info: "page reloaded"}
 }
+
+// MessageExport simulates the given user performing
+// a compliance message export
+func MessageExport(u user.User) UserActionResponse {
+	isAdmin, err := u.IsSysAdmin()
+	if err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
+
+	if !isAdmin {
+		return UserActionResponse{Info: "user is not a sysadmin and cannot perform a message export"}
+	}
```

Comment:
Every user is calling this action but only users that are admins will proceed with actually triggering a message export. We have a similar check in the `Reload` action. Happy to bubble this up to the actual action invocation.

Reply:
The difference is that the Reload case is somewhat optional, here the whole action needs admin permission. I am mainly trying to avoid a spam of logs otherwise it's the same really.